src/Database/TigerBeetle/Raw/Client.hs

```diff
@@ -3,7 +3,14 @@
 {-# LANGUAGE LambdaCase #-}
 {-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
 
-module Database.TigerBeetle.Raw.Client where
+module Database.TigerBeetle.Raw.Client (
+  submitRequest,
+  finalizeClient,
+  withClient,
+  ClientConfig(..),
+  defaultConfig,
+  ClientHandle(..),
+) where
```

Comment:
Is this a weeder change because it requires us to use export lists on all modules?